PriceQuoteIos/Utils/APIHandler.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  APIHandler.swift
+//  PriceQuoteIos
+//
+//  Created by Livia Vasconcelos on 27/09/20.
+//  Copyright © 2020 Livia Vasconcelos. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+class APIHandler {
```

Comment:
For this project, is this error handling structure enough?

Reply:
I changed a little bit since I don't have Alamofire anymore.